src/code/circuit_python/motor.md

```diff
@@ -0,0 +1,232 @@
+
+API documentation for the `pwmio` module is available
+[here](https://docs.circuitpython.org/en/latest/shared-bindings/pwmio/index.html).
+
+## The adafruit_motor Module
+
+The `adafruit_motor` module provides tools for controlling a variety of
+types of motors. We will use the `adafruit_motor.servo` module specifically.
+
+Why would we use a "servo" module for controlling our motors? In our case,
+it's because they use the same control interface. The Vex MC29 makes our
+DC gear motors behave like "continuous rotation servos" by using PWM to
+control them. You've probably seen the common three-wire connector used by
+the Vex MC29 and servos in your kit. The red and black wires provide power
+to the servo/motor and the white wire carries a PWM signal to control the
+servo / motor.
+
+Servos control their position based on the signal they receive. We can
+control servos using the `Servo` class in `adafruit_motor.servo`. It has a
+property called `angle` that lets us set the position of the servo.
+Another property called `actuation_range` tells the `Servo` object how far
+the physical servo is able to move, so the code knows how to convert our
+angles into PWM signals.
+
+For controlling motors, we can use the `ContinousServo` class in
+`adafruit_motor.servo`. It has a property called `throttle` which controls
+the speed of the motor. `throttle` can be set to any value between -1.0
+and 1.0. Positive numbers will make the motor spin one direction and
+negative numbers will make the motor spin the other direction. Zero
+throttle will stop the motor.
+
+Both of these objects are created by giving them a `PWMOut` object
+connected to a pin. Optionally, you can also tell them the minimum and
+maximum pulse widths in milliseconds.
+
+API documentation for the `adafruit_motor` module is available
+[here](https://docs.circuitpython.org/projects/motor/en/latest/api.html).
+
+
+## Looping Over Numbers
+
+A common need in programs is looping over a range of numbers. This could
+be just because we need to repeat some action a certain number of times,
+or to loop through the elements in a list of items, for example. In
+Python, we can do this with a `for` loop and the `range` function.
+
+A `for` loop, in Python, runs the body of its loop once for each element a
+container. If you give it a list of numbers like `[1, 2, 3, 4]`, the loop
+will run four times.
+
+```Python
+# This loop will run 4 times
+for i in [1, 2, 3, 4]:
+  # i holds the current element from the list
+  print(i)
+# This code will output:
+# 1
+# 2
+# 3
+# 4
+```
+
+Usually, we don't want to list out each number in the range one by one.
+Instead, we create a `range` object. A `range` object can be created in
+one of three ways:
+
+```Python
+# Create a range with the number of values it should hold
+# This range will hold the numbers 0 through 9
+range(10)
+
+# Create a range with a start and stop value
+# This range will hold the numbers 5 through 19
+range(5,20)
```

Comment:
Perhaps note here that the start is inclusive and the stop is exclusive?  You call this out later, but it would be good to add a reason here as well.